File: quickstarts/VB/EventBroker/Tests/Tests.EventBrokerExtension/TestObjects/OneEventPublisher.vb
' Copyright (c) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.

Imports System
Imports SimpleEventBroker

Namespace Tests.EventBrokerExtension.TestObjects
	Class OneEventPublisher
    <Publishes("paste")> _
  Public Event Pasting As EventHandler
	End Class
End Namespace

File: quickstarts/CS/EventBroker/Tests/Tests.EventBrokerExtension/TestObjects/OneEventPublisher.cs
// Copyright (c) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.

using System;
using SimpleEventBroker;

namespace Tests.EventBrokerExtension.TestObjects
{
    class OneEventPublisher
    {
#pragma warning disable 67 // Disable "Event is never used" warning
        [Publishes("paste")]
        public event EventHandler Pasting;
    }
}
